examples/staking/staking.py

```diff
@@ -0,0 +1,154 @@
+import json
+import sys
+import time
+from pathlib import Path
+
+from multiversx_sdk_core import AddressFactory, TokenComputer, TransactionComputer, Address
+from multiversx_sdk_core.transaction_factories import TransactionsFactoryConfig, SmartContractTransactionsFactory, \
+    TransferTransactionsFactory
+from multiversx_sdk_network_providers import ProxyNetworkProvider
+from multiversx_sdk_network_providers.transactions import TransactionOnNetwork
+from multiversx_sdk_wallet import UserPEM, UserSigner
```

Comment:
suggestion to add a `requirements.txt` file for the dependencies and maybe a separate README here with some info on how to run; it can be done also in a future PR